openedx/core/djangoapps/user_api/accounts/tests/pact/PactBypassAuthMiddleware.py

```diff
@@ -0,0 +1,36 @@
+"""
+Contain the middleware logic needed during pact verification
+"""
+from django.contrib import auth
+from django.utils.deprecation import MiddlewareMixin
+from lms.envs.pact import PACT_BYPASS_AUTHENTICATION
```

Comment:
`lms.envs.pact` acts as Django settings module, so rather than importing it directly, you'll want to use the Django settings mechanism. This allows the user to select which set of settings is in effect.

To use it, you'll need `from django.conf import settings` in the imports. `settings` is a dictionary-like object, so you can use `getattr` to read from it.

Here's an example of some code reading a Django setting and documenting it: https://github.com/openedx/event-bus-kafka/blob/0e86844bd923a30abcd1cff29b331708eae13fea/edx_event_bus_kafka/internal/config.py#L60-L64

Reply:
Okay great, I'll use the `getattr` method to get the variable.